src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/commands.py

```diff
@@ -44,21 +47,26 @@ def _dont_fail_not_exist(ex):
         raise ex
 
 
+transform_keys_result = lambda x: x.keys if hasattr(x, 'keys') else x  # noqa: E731 lambda vs def
```

Comment:
PEP8:
Always use a def statement instead of an assignment statement that binds a lambda expression directly to an identifier.

Yes: def f(x): return 2*x
No: f = lambda x: 2*x